server/routerlicious/packages/lambdas/src/nexus/connect.ts

```diff
@@ -92,6 +93,9 @@ function composeConnectedMessage(
 		initialMessages: [],
 		initialSignals: [],
 		supportedVersions: ProtocolVersions,
+		supportedFeatures: {
+			submit_signals_v2: true,
```

Comment:
If we end up having signals v3 what happens? Would we have more flexibility if we expressed signals versions more explicitly as "v2", "v3" etc?

Or maybe consider just state "targeted_signals: tree" and get away from versioning? When you bring up versioning bunch of doors open up, like is it min version or actual supported? what set of features individual version represents etc.

Reply:
I'm not aware of the future vision of v2 signals but it looks like it includes more than just the ability to target clients. For example, looking at the new signal type [ISentSignalMessage](https://github.com/microsoft/FluidFramework/blob/main/common/lib/protocol-definitions/src/protocol.ts#L337-L382), client code can specify a referenceSequenceNumber that indicates when signal was created to data stream. Currently nothing is done with this but once implemented it would fall under v2 functionality.

I think renaming to "targeted_signals: true" would be fine, we would just have to continue renaming/adding more supported features as more functionality is added.

Reply:
It looks like the main reasoning here is that we are offering similar functionality through ODSP (leveraging Vx signals schema), and we want to keep the types similar, or the same?